zebra-rpc/src/methods/get_block_template_rpcs/config.rs

```diff
@@ -0,0 +1,13 @@
+//! Mining config
+
+use serde::{Deserialize, Serialize};
+
+use zebra_chain::transparent::Address;
+
+/// Mining configuration section.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[serde(deny_unknown_fields, default)]
+pub struct Config {
+    /// Miner transparent address
+    pub miner_transparent_address: Option<Address>,
```

Comment:
If we want to support shielded addresses later, we might want to call this a name that will work for both.
(There is only ever one miner address.)

```suggestion
    pub miner_address: Option<Address>,
```